debian/control

```diff
@@ -0,0 +1,30 @@
+Source: revpi-sos-report
+Section: electronics
+Priority: optional
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Build-Depends:
+  debhelper-compat (= 13),
+  dh-python,
+  python3-dev,
+  cmake
+Standards-Version: 4.5.1
+Homepage: https://github.com/RevolutionPi/revpi-sos-report
+Vcs-Browser: https://github.com/RevolutionPi/revpi-sos-report/tree/raspios/bullseye
+Vcs-Git: https://github.com/RevolutionPi/revpi-sos-report.git -b raspios/bullseye
+Rules-Requires-Root: no
```

Comment:
We don't have any templates for our control files, but in the last changes in some repositories we are using something like that: [See this commit](https://github.com/RevolutionPi/noderedrevpinodes-server/pull/4/commits/2dc0acb3e0f7b646d7e6e1334f4436c693c87f33)

I think that we can use the same structure here... What do you think?